src/Controller.js

```diff
@@ -275,19 +322,8 @@ export default class Controller {
 	}
 
 	_listenCallback() {
```

Comment:
I added these checks while testing a [three.js example](https://threejs.org/examples/#webgl_clipping_advanced) that had an uncheckable checkbox with `listen()`. I thought `listen()` was causing the checkbox to lose interactivity by updating the display every frame. The author's intent was actually to disable the controller by using a setter that could ignore assignment calls. 

This code appeared to "fix the bug" because the value of `input.checked` would be allowed to change, while in truth it was out of sync with `getValue()`, and was never forced to `updateDisplay()` because the never changed between frames.